### ISSUE TITLE
about result of critical node

### ISSUE THREAD

**Issue by GLHFGGLFZ** [FFrankyy/FINDER#35]
Hello, I am currently using FINDER to process my data. 
I would like to ask about the node label sorting in the result file (for example, the Crime_degree.txt file) to determine which nodes are important. 
What do the values in the MaxCCList__Strategy_Crime file represent? Do they represent maximum Closeness Centrality? Or something else?I couldn't find the specific meaning of MaxCCList in the literature and supplementary materials.
In summary, how can I determine the importance ranking of nodes from these two files?

**Comment by chenshuai17** [comment]
Do you know how to interpret the results of these two files now?
